src/bulma.tsx

```diff
@@ -259,15 +245,12 @@ export function removeLoadingProps(props: Bulma.Loading) {
     return rest;
 }
 
-export function getColorModifiers({ isColor: color }: Bulma.Color) {
-    return isColor(color) ? { [`is-${color}`]: true } : {};
+export function getColorModifiers(color: Bulma.Color) {
+    return { [`is-${color}`]: true };
 }
 
 export function removeColorProps(props: Bulma.Color) {
```

Comment:
Yes, we need to remove this function, also we need to remove it from all the components that are using this. For example, [Button](https://github.com/AlgusDark/bloomer/blob/02711d305295d5936ff5c7ae50a6317a6a071812/src/elements/Button.tsx#L7) is using it.

Reply:
So this spreads the array of props, and pulls out the one called `isColor`, then discards it by returning `rest`. I didn't understand what it was doing at first reading.. pretty neat syntax 🙂

Reply:
That's correct, that's what we do, so we pass only valid `DOM props` to the element. That way we don't try to render something like `<div isColor="is-white"></div>` since that's not valid DOM.